### PR TITLE
allow excluding certain gems from auto-updates

### PR DESCRIPTION
**Pull request by evgeni**
ostree_client is too special

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
We already have `is_nightly` logic to skip nightlies. Isn't that a better place? So possibly implement something like `is_updateable` on `Spec` that is `self is_nightly or ...`.

**Comment by evgeni** [comment]
Good idea!

**Comment by evgeni** [comment]
Updated!

File: list_updatable_packages [hunk 1]
```diff
@@ -12,6 +12,7 @@ import requests
 import tempfile
 
 SUPPORTED_TEMPLATES = ('foreman_plugin', 'hammer_plugin', 'smart_proxy_plugin')
+EXCLUDED_GEMS = ('pulp_ostree_client',)
```
Comment (ekohl) [inline list_updatable_packages:15]:
```suggestion
# Only prereleases exist and our tooling can't handle that
EXCLUDED_GEMS = ('pulp_ostree_client',)
```

**Review by ekohl — APPROVED** [review]

**Pull request merged**